### PR TITLE
Adds "destination" parameter which allows uploading files to a different location.

### PR DESCRIPTION
**Pull request by mhaamann**
TODO: 

- [x]  Fix prune logic so it does not fail the build if the directory does not exist. This is especially a problem on the first run.

### CODE REVIEW

**Comment by ayeressian** [comment]
Hi @mhaamann , thank you for the contribution.
A question, can't we change the code to also support nested directories?

**Comment by ayeressian** [comment]
Does the directory need to exist before uploading or it's going to create it before upload?

**Comment by mhaamann** [comment]
>A question, can't we change the code to also support nested directories?
I believe it already supports this. I will test.

>Does the directory need to exist before uploading or it's going to create it before upload?
The directory gets created automatically by Bunny when uploading.

**Comment by mhaamann** [comment]
- Added some tests to the remove function.
- I confirmed that uploading nested directories works
- I confirmed that removing nested directories works. 
- Ensured that the remove function does not fail the build when the destination does not exist.

File: README.md [hunk 1]
```diff
@@ -25,6 +25,11 @@ It will purge the pull zone if "true" provided. pullZoneId and accessKey inputs
 
 The source directory that should be uploaded.
 
+### `destination`
+
+The destination directory that should be uploaded too in the bunny storage zone. (Example: www).
+The destination should not end with `/` as in `www/`.
```
Comment (ayeressian) [inline README.md:31]: Can you add an example of nested directory as well?

**Review by ayeressian — COMMENTED** [review]
LGTM. One minor thing, can you add an example of nested directory in README?

File: README.md (hunk 1 above)
Comment (mhaamann) [reply README.md]: Sure, just added an example. Is this what you mean?

Comment (ayeressian) [reply README.md]: What I meant was for example "dist/release1/draft". Do we support this? Will it create all the 3 directories?

**Comment by ayeressian** [comment]
@mhaamann I will push a new release shortly

**Comment by ayeressian** [comment]
@mhaamann your contribution is available on v2.2.0 release. Thanks!

**Pull request merged**